01-node-tutorial/answers/01-intro.js

```diff
@@ -0,0 +1,10 @@
+const city = 'Berlin';
+console.log(city);
+
+const country = '';
+
+if (country) {
+  console.log(country);
+} else {
+  console.log('country does not exist');
+}
```

Comment:
This looks good, another option would be:

`if (!country) {
  console.log('country does not exist)
 }
 
 console.log(country)
`